### PR TITLE
TSPS-280 e2e test fix - auto-pull lastest git release tag if not specified

### PR DESCRIPTION
**Pull request by mmorgantaylor**
### Description 

Successful run without specifying a wdl method version: https://github.com/DataBiosphere/terra-scientific-pipelines-service/actions/runs/10948530034

### Jira Ticket
https://broadworkbench.atlassian.net/browse/TSPS-280

### CODE REVIEW

File: .github/workflows/run-e2e-tests.yaml [hunk 1]
```diff
@@ -15,46 +15,47 @@ on:
...
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Get inputs or use defaults, generate other parameters
+        id: extract-inputs-and-generate-params
         run: |
-          echo "delete-bee=${{ inputs.delete-bee || true }}" >> "$GITHUB_OUTPUT"
+          echo "delete-bee=${{ inputs.delete-bee || true }}" >> $GITHUB_OUTPUT
           if [ -z ${{ inputs.custom-app-version }} ]; then
-            echo "e2e-test-version-ref={}" >> "$GITHUB_OUTPUT"
```
Comment (mmorgantaylor) [inline .github/workflows/run-e2e-tests.yaml:37]: this was just wrong before, we'd accidentally wired custom-app-version through e2e-test-version-ref.

Comment (jsotobroad) [reply .github/workflows/run-e2e-tests.yaml:37]: nice catch

**Review by jsotobroad — APPROVED** [review]
sweet, looks clean.  Am I correct with assuming this will be the pulling the tag of the previous release for the wdl and not like the one that is actively being generated as part of this test running (which makes total sense)?

**Comment by mmorgantaylor** [comment]
@jsotobroad yes, this will grab the latest published version, so it won't catch any changes in a PR until merged.

**Pull request merged**